src/test/features/controls-spec.ts

```diff
@@ -361,6 +361,38 @@ suite('ModelViewerElementBase with ControlsMixin', () => {
           expect(promptElement.classList.contains('visible'))
               .to.be.equal(false);
         });
+
+        test('plays css animation only when visible', async () => {
+          element.interactionPrompt = 'auto';
+
+          const computedStyle =
+              getComputedStyle((element as any)[$promptAnimatedContainer]);
+          expect(computedStyle.animationPlayState)
+              .to.be.match(/^paused(\, paused)?$/);
+
+          await timePasses(element.interactionPromptThreshold + 100);
```

Comment:
This doesn't seem to directly test visibility.

Reply:
Will fix